frontend/src/Components/DescriptionList/DescriptionListItem.js

```diff
@@ -18,7 +18,7 @@ class DescriptionListItem extends Component {
     } = this.props;
 
     return (
-      <span className={className}>
+      <div className={className}>
```

Comment:
aria related or something else?

Reply:
HTML spec actually, span are not valid inside of there